packs/opentelemetry_collector/templates/opentelemetry_collector.nomad.tpl

```diff
@@ -0,0 +1,97 @@
+[[- $vars := .opentelemetry_collector -]]
+job [[ template "full_job_name" . ]] {
+  [[ template "region" . ]]
+
+  datacenters = [[ $vars.datacenters | toPrettyJson ]]
+  namespace   = [[ $vars.namespace | quote ]]
+
+  type = [[ $vars.job_type | quote ]]
+
+  [[ if $vars.constraints ]][[ range $idx, $constraint := $vars.constraints ]]
+  constraint {
+    attribute = [[ $constraint.attribute | quote ]]
+    value     = [[ $constraint.value | quote ]]
+    [[- if ne $constraint.operator "" ]]
+    operator  = [[ $constraint.operator | quote ]]
+    [[- end ]]
+  }
+  [[- end ]][[- end ]]
+
+  group "opentelemetry_collector" {
+    [[- if eq $vars.job_type "service" ]]
+    count = [[ $vars.instance_count ]]
+    [[- end ]]
+    network {
+      mode = [[ $vars.opentelemetry_collector_group_network.mode | quote ]]
+      [[- range $label, $to := $vars.opentelemetry_collector_group_network.ports ]]
+      port [[ $label | quote ]] {
+        to = [[ $to ]]
+      }
+      [[- end ]]
+    }
+
+    task "opentelemetry_collector" {
+      driver = "docker"
+
+      config {
+        image = "[[ $vars.opentelemetry_collector_task_config.image ]]:[[ $vars.opentelemetry_collector_task_config.version ]]"
+        args = [[ $vars.opentelemetry_collector_task_config.cli_args | toPrettyJson ]]
+        [[- if $vars.privileged_mode ]]
+        pid_mode = "host"
+        [[- end ]]
+        volumes = [
+          "local/otel:/etc/otel",
```

Comment:
I suspect this needs to be a configurable host volume.

Reply:
Ah! Thank you!

I'll get this sorted by end of week.

Reply:
@DerekStrickland did a bit of a re-work of all of this. I think it wasn't running because the defaults weren't smart -- I've fixed that and provided two examples of how to use the Pack along with some docs update.